crates/ika-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -548,43 +427,7 @@ impl DWalletMPCManager {
     /// Spawns all ready MPC cryptographic computations using Rayon.
     /// If no local CPUs are available, computations will execute as CPUs are freed.
     pub(crate) async fn perform_cryptographic_computation(&mut self) {
-        for event in self
-            .events_pending_for_network_key
-            .drain(..)
-            .collect::<Vec<_>>()
-            .into_iter()
-        {
-            self.handle_dwallet_db_event(event.clone()).await;
-        }
```

Comment:
I mean this is really funny, we shouldn't just try and retry and retry until we succeed. There should be a succeess criterion we know how to check, and we should know if something should fail or succeed in advance of doing it, and not just do random things and hope for the best

Reply:
Also, this event handling logic has nothing to do with `perform_cryptographic_computation`